build_info/sl.control

```diff
@@ -0,0 +1,8 @@
+Package: sl
+Version: @DEB_SL_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Depends: ncurses
+Section: Utilities
+Priority: standard
```

Comment:
Pet peeve: This should be `Priority: optional`.

Reply:
Should be fixed now